modules/services/muchsync.nix

```diff
@@ -0,0 +1,160 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.muchsync;
+in {
+  meta.maintainers = with maintainers; [ pacien ];
+
+  options.services.muchsync = {
+    enable = mkEnableOption "muchsync periodic sync";
+
+    frequency = mkOption {
+      type = types.str;
+      default = "*:0/5";
+      description = ''
+        How often to run muchsync. This
+        value is passed to the systemd timer configuration as the
+        <literal>OnCalendar</literal> option. See
+        <citerefentry>
+          <refentrytitle>systemd.time</refentrytitle>
+          <manvolnum>7</manvolnum>
+        </citerefentry>
+        for more information about the format.
+      '';
+    };
+
+    server = mkOption {
+      type = types.str;
+      description = ''
+        Remote SSH host to synchronize with.
+      '';
+    };
+
+    configFile = mkOption {
+      type = types.str;
+      default = "";
+      defaultText = "$NOTMUCH_CONFIG";
+      description = ''
+        Specify the path of the notmuch configuration file to use.
+        If none is specified, the default is to use the contents of the environment variable
+        $NOTMUCH_CONFIG, or if that variable is unset, the value $HOME/.notmuch-config.
+        (These are the same defaults as the notmuch command itself.)
+      '';
+    };
+
+    checkForModifiedFiles = mkOption {
+      type = types.bool;
+      default = false;
+      description = ''
```

Comment:
These descriptions seem to be copied? Is there any licensing issues to consider?

Descriptions should be in docbook format and paragraph breaks have to be marked by `</para><para>`.

Also, the descriptions need to be fixed to make sense for a Home Manager user. For example, `-F` will never be seen by the HM user so it seems unpedagogical to refer to it.

Try to keep line widths below 80 characters.

Reply:
> These descriptions seem to be copied? Is there any licensing issues to consider?

The project and its documentation are published under the GPL v2+.
I added a comment with a link to the original manual page at the top of the module.

> Descriptions should be in docbook format and paragraph breaks have to be marked by `</para><para>`.
> 
> Also, the descriptions need to be fixed to make sense for a Home Manager user. For example, `-F` will never be seen by the HM user so it seems unpedagogical to refer to it.
> 
> Try to keep line widths below 80 characters.

Fixed.